### PR TITLE
Feat/68/Add_division_view

### PR DESCRIPTION
**Pull request by Zhykovskiy**

### CODE REVIEW

**Review by vladtymo — CHANGES_REQUESTED** [review]
Let's use the reactive form.

File: MilitaryManager/client-app/src/app/shared/services/core/http.service.ts
Comment (vladtymo) [inline MilitaryManager/client-app/src/app/shared/services/core/http.service.ts]: What was the reason to make these changes?

File: MilitaryManager/client-app/src/app/modules/division-module/pages/division-new/division-new.component.ts
Comment (vladtymo) [inline MilitaryManager/client-app/src/app/modules/division-module/pages/division-new/division-new.component.ts]: Where do you use this property?

File: MilitaryManager/client-app/src/app/shared/services/core/http.service.ts
Comment (Zhykovskiy) [reply MilitaryManager/client-app/src/app/shared/services/core/http.service.ts]: These are changes from branch 78. I'll wait for its merge before my merge.

File: MilitaryManager/client-app/src/app/modules/division-module/pages/division-new/division-new.component.ts
Comment (Zhykovskiy) [reply MilitaryManager/client-app/src/app/modules/division-module/pages/division-new/division-new.component.ts]: Deleted it.

**Review by vladtymo — APPROVED** [review]

**Review by Alex-knu — APPROVED** [review]

**Pull request merged**
